script/sem-ver-action.py

```diff
@@ -0,0 +1,70 @@
+#  imports
```

Comment:
Please add a logger to the relevant places. Make sure you are using the appropriate log levels (info, debug, and error) where necessary.

Reply:
added!